src/status_im/ui/components/list/styles.cljs

```diff
@@ -150,7 +138,7 @@
    :margin-left  6
    :margin-right 4
    :font-size    11
-   :font-weight  :bold})
+   :font-weight  700})
```

Comment:
so, wondering here, does it work without curly quotes? I thought RN specified that font-weight should be specified like '500', '600', '700' etc?

Reply:
because in typography a font-family is selected based on font-weight and font-style